src/app/_components/Card.tsx

```diff
@@ -55,16 +55,11 @@ function CardImage({ image }: Pick<CardProps, 'image'>) {
   const isStaticImage = 'data' in image
```

Comment:
is this needed? do we use static images in Card component?

Reply:
We do in the `Reports` on the about page